ethereum_history_api/oracles/src/script/prepareMerkleProofFixtures.ts

```diff
@@ -0,0 +1,45 @@
+import { mkdir, writeFile } from 'fs/promises';
+import { createMerkleProofFixture } from './noir_fixtures/merkle_proof.js';
+import { Trie } from '@ethereumjs/trie';
+import { PROOF_FIXTURES } from '../fixtures/merkleProofsConfig.js';
+import { assert, encodeHexStringToArray } from '../main.js';
+import { Hex } from 'viem';
+
+const NOIR_PROOF_FIXTURES_DIRECTORY = '../circuits/lib/src/fixtures/merkle_proofs';
+
+function hasDuplicates(arr: Hex[]) {
+  return new Set(arr).size !== arr.length;
+}
+
+let fixtureModule = ``;
+const fixtureModuleFile = `${NOIR_PROOF_FIXTURES_DIRECTORY}.nr`;
+
+for (const fixtureName in PROOF_FIXTURES) {
+  await mkdir(NOIR_PROOF_FIXTURES_DIRECTORY, { recursive: true });
+
+  const keyValuePairs = PROOF_FIXTURES[fixtureName].keyValuePairs;
+  assert(
+    !hasDuplicates(keyValuePairs.map((keyValuePair) => keyValuePair.key)),
```

Comment:
Try `Object.keys(keyValuePairs)`

Reply:
When I replace it with .keys() I get a following error:
Argument of type 'IterableIterator<number>' is not assignable to parameter of type 'unknown[]'.
  Type 'IterableIterator<number>' is missing the following properties from type 'unknown[]': length, pop, push, concat, and 28 more.